cmake/SndFileChecks.cmake

```diff
@@ -213,6 +213,13 @@ if (MSVC)
 	add_definitions (-D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE)
 endif (MSVC)
 
+if (MINGW)
+	set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -static-libgcc")
+	set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -static-libgcc -static-libstdc++")
+	set (CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_C_FLAGS} -static-libgcc -s")
+	set (CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS} -static-libgcc -static-libstdc++ -s")
+endif ()
```

Comment:
@evpobr Maybe I should open an issue. I would like to know why static link with libgcc and libstc++ is forced? If there is no special reason, this decision should be left to consumers.

Reply:
And it is done: libsndfile/libsndfile#630.